configs/FLYCOLORF435/config.h

```diff
@@ -0,0 +1,117 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU        AT32F435G
+
+#define BOARD_NAME           FLYCOLORF435
+#define MANUFACTURER_ID      FLCO
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
```

Comment:
Missing gyro clock in define and timer assignment

Reply:
> Missing gyro clock in define and timer assignment
I have made the following settings
1,USE_GYRO_CLKIN
2, GYRO_1_CLKIN_PIN PC5 //(TMR9_CH2)
3, TIMER_PIN_MAP( 2, PC5, 1, -1)
But I can't see the clock signal on the CLKIN pin, can't TMR9_CH2 be used as an external clock for GYRO

Reply:
> Are you building BF 4.6? Post a support ID.
It can work now.
I used 4.6 a while ago, but it didn't work So switching to 4.5-maintenance still didn't work. Today, I tested with 4.6 and found a 32KHZ signal at the CLKIN pin.